descriptor/qsarmolecular/src/main/java/org/openscience/cdk/geometry/surface/Point_Type.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2004-2007  The Chemistry Development Kit (CDK) project
+ *
+ * Contact: cdk-devel@lists.sourceforge.net
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * as published by the Free Software Foundation; either version 2.1
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package org.openscience.cdk.geometry.surface;
+
+import javax.vecmath.Point3d;
+
+
+public class Point_Type
```

Comment:
I personally prefer it to have no hyphen in the class name.
Also, please add JavaDoc to all public classes and methods.

Reply:
No new class needed, this is a simple pair, the method return type should be ``List<Map.Entry<IAtom,Point3d>>``.